internal/storage/storage_handle.go

```diff
@@ -31,8 +31,8 @@ type StorageHandle interface {
 	BucketHandle(bucketName string) (bh *bucketHandle, err error)
 }
 
-type storageClient struct {
-	client *storage.Client
+type Storageclient struct {
```

Comment:
do we still need this to be public

Reply:
Yes because we are returning storageclient object in fake_storage_util and receiving it in bucket_manager_test

Reply:
fixed